### ISSUE TITLE
amp-accordion not working on IE11 & Edge

### ISSUE THREAD

**Issue by cloudb** [ampproject/amphtml#23254]
## What's the issue?

As of sometime today the amp accordion component is no longer working in IE11 and Edge. 
The following error appears in the console.

`failed to build: amp-accordion#24 SyntaxError`

Assume it's related to [this](https://github.com/ampproject/amphtml/pull/22755) PR which allows 'bind on [data-expand]' but also appears to have amended the integration test to skip Edge.

## How do we reproduce the issue?

Visit [amp-accordion - amp by example](https://amp.dev/documentation/examples/components/amp-accordion/?referrer=ampbyexample.com) or any page with amp accordion component on IE11 or Edge.

## What browsers are affected?

IE11 and Edge

## Which AMP version is affected?

Latest version.

**Comment by robinac30** [comment]
"Assume it's related to this PR which allows 'bind on [data-expand]' but also appears to have amended the integration test to skip Edge." - that's worrying - this is a supported browser no?

**Comment by gilbertococchi** [comment]
@rudygalfi @aghassemi Can you please take a look at this issue?

Issue is reproducible by browsing https://amp.dev/documentation/examples/components/amp-accordion/preview/?format=websites on Desktop Edge.

Screenshot below

![Screenshot 2019-07-12 at 10 39 09](https://user-images.githubusercontent.com/329023/61114807-66000000-a491-11e9-94b7-7ce9332e7950.png)

**Comment by aghassemi** [comment]
@kristoferbaxter, this is a regression. Likely due to use of Mutation observer in the PR reference above. Regression is usually treated as somewhere between P0 and P1. I suggest patching the canary that is going to prod next week with a fix.

**Comment by cloudb** [comment]
Is this going to be resolved in time for the upcoming release?  If this is somewhere between a P0 and P1 can it be labelled as so to ensure someone takes a look at it?

**Comment by nainar** [comment]
Hi folks, 

I have marked this issue as a P0. Sorry this has gotten to prod pages. 

@cvializ could you provide an update here when you have a handle on the issue? Thank you!

**Comment by cvializ** [comment]
Found root cause and opened PR #23328
